Services/Mail/classes/AutoResponder/class.ilAutoResponder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
```

Comment:
The license header is wrong. I know that `PhpStorm` automatically formats it, but this is wrong.